packages/component-library/src/components/form/mt-password-field/mt-password-field.vue

```diff
@@ -22,7 +23,8 @@
 
       <button
         v-if="toggable"
-        @click="showPassword = !showPassword"
+        type="button"
```

Comment:
Why is type button needed here? It should not be necessary nor will this change the semantics for Screen Readers as it is already a button 🤔

Reply:
If your buttons are not for submitting form data to a server, be sure to set their type attribute to button . Otherwise, they will try to submit form data and to load the (nonexistent) response, possibly destroying the current state of the document.